17.0.Dockerfile

```diff
@@ -36,7 +36,7 @@ ENV DB_FILTER=.* \
 RUN apt-get -qq update \
     && apt-get install -yqq --no-install-recommends \
         curl \
-    && curl -SLo wkhtmltox.deb https://github.com/wkhtmltopdf/wkhtmltopdf/releases/download/${WKHTMLTOPDF_VERSION}/wkhtmltox_${WKHTMLTOPDF_VERSION}-1.buster_amd64.deb \
+    && curl -SLo https://github.com/wkhtmltopdf/packaging/releases/download/${WKHTMLTOPDF_VERSION}-3/wkhtmltox_${WKHTMLTOPDF_VERSION}-3.bookworm_amd64.deb \
```

Comment:
did you test if the Odoo S.A. maintained version of wkhtmltopdf works here?
https://github.com/odoo/wkhtmltopdf

Would be nice if we can switch to the one built for Odoo.

Reply:
Oh, okay thank you! I am going to change to the odoo wkhtmltopdf nightly build

Reply:
Other option is to use dockerized wkhtmltopdf: https://github.com/acsone/kwkhtmltopdf. That way, we also isolate possible security issues.